ui/components/app/wallet-overview/token-overview.test.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { generateUseSelectorRouter } from '../../../hooks/gasFeeInput/test-utils';
+import { renderWithProvider } from '../../../../test/jest/rendering';
+import TokenOverview from './token-overview';
+
+jest.mock('react-redux', () => {
+  const actual = jest.requireActual('react-redux');
+
+  return {
+    ...actual,
+    useSelector: jest.fn(),
+  };
+});
+
+describe('TokenOverview', () => {
+  const mockStore = {
+    metamask: {
+      provider: {
+        type: 'test',
+      },
+      preferences: {
+        useNativeCurrencyAsPrimaryCurrency: true,
+      },
+    },
+  };
+
+  const store = configureMockStore([thunk])(mockStore);
+
+  beforeAll(() => {
```

Comment:
Why is this a `beforeAll`, should this be a `beforeEach`?

Reply:
beforeEach probably makes more sense!

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/13291/commits/4139652022a579b67556f3783bfbea1ddfc4e609